facility-all-programs.ipynb

```diff
@@ -424,16 +572,16 @@
     "        have_water_enf = True\n",
     "        # print( water_enf_data )\n",
     "    except pd.errors.EmptyDataError:\n",
-    "        print( \"No data for this facility in NPDES_FORMAL_ENFORCEMENT_ACTIONS.\")\n",
+    "        print( \"This is a CWA-regulated facility, but there's no data for it in NPDES_FORMAL_ENFORCEMENT_ACTIONS.\")\n",
     "    "
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Show the information from one of the data sets for this facility\n",
-    "Running this cell will create a dropdown list of the program data sets available for this faciliity. Pick one and then run the second cell to view its contents. To see additional data, return to the dropdown to choose a new value and repeat."
```

Comment:
It makes sense to force selection of location & facility, because there are so many, but is there any reason not to just run the graph for all the available datasets, since there can only be a handful of them? More streamlined user experience & would let people see side by side

Reply:
I think that's a great idea. My hesitations: 1) having something ready for Chris's class by Thursday, and this is what we have; 2) more importantly, there can be up ~10 charts, which may be overwhelming in the output space. There may be good ways of displaying multiple charts at a time (subplots https://stackoverflow.com/questions/22483588/how-can-i-plot-separate-pandas-dataframes-as-subplots), but...see 1)

Reply:
Yeah, makes sense. We can revisit later as we keep developing